### ISSUE TITLE
Customizing training data for fine-tuning the model

### ISSUE THREAD

**Issue by sumin5784** [jerryji1993/DNABERT#59]
Hello,

I'm trying to use my own dataset to fine-tune DNABERT-5/6.
I have several questions about this.

1. All input sequence length should be the same? Or input sequences can be different length?
2. Does label should be 1/0? Basically, does classification class can be more than two classes?

Any feedback would be appreciated.
Thank you!

**Comment by Leven1113** [comment]
> Excuse me, do you understand these two questions？

**Comment by OPUS-Lightphenexx** [comment]
yes, i was wondering whether the labels could be 1 to 10 rather than binary classify task
